src/main/java/org/folio/linked/data/util/BibframeUtils.java

```diff
@@ -85,6 +90,9 @@ public static List<Resource> extractInstances(Resource resource) {
     if (resource.isOfType(INSTANCE)) {
```

Comment:
minor
for me name of this method in't clear
this method can actually work only with Work and Instance
but if we have
Instance - > Work - >MainTitle
I expected that
`extractInstances(Resource mainTitle)` // << mainTitle
can return `Resource( Instance)`  from this graph

the same example in method upper `extractWork(Resource resource)`